package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-useasynccall",
```

Comment:
co to pojmenovat `react-async-call-hook`? repo taky

Reply:
nebo podle https://react-hook-form.com/ `react-hook-async-call`?

Reply:
react-hook-async-call se mi líbí víc :)